external/anomaly/anomaly_classification/openvino.py

```diff
@@ -99,6 +100,8 @@ def __init__(self, task_environment: TaskEnvironment) -> None:
         labels = task_environment.get_labels()
         self.normal_label = [label for label in labels if label.name == LabelNames.normal][0]
         self.anomalous_label = [label for label in labels if label.name == LabelNames.anomalous][0]
+        template_file_path = task_environment.model_template.model_template_path
```

Comment:
maybe we could rename `template_file_path` to `model_template_path` for consistency